### PR TITLE
Adding minutes for March 6

### LINKED ISSUE #37: Node.js i18n WG Meeting 2018-03-06
## Time
UTC Tue 06-March-2018 16:00 (04:00 PM)
[What is 4PM-UTC In your local time?](http://www.wolframalpha.com/input/?i=04PM+UTC,+March+06,+2018+in+local+time)

## Links
* [Minutes Google Doc](https://docs.google.com/document/d/1daZuYpjomqVylkpNJ2TDLa3Gjd-3KvrNpMBkpwxNlOQ)

## [Current Members](https://github.com/nodejs/i18n#current-members)
@amiller-gh (Adam Miller)
@toinane (Antoine Olivier)
@obensource (Ben Michel)
@tiriel (Benjamin Zaslavsky)
@maddhruv (Dhruv Jain)
@JCMais (Jonathan Cardoso Machado)
@lukaszewczak (Łukasz Szewczak)
@rachelnicole (Rachel White)
@rajzshkr (Raja Sekar)
@RichardLitt (Richard Littauer)
@ryo-a (Ryo.A)
@sotayamashita (Sam Yamashita)
@srl295 (Steven R. Loomis)
@tiagoDanin (Tiago Danin)
@vanessayuenn (Vanessa Yuen)
@volem (Volkan Nazmi Metin)
@zeke (Zeke Sikelianos)

### General Logistics
* [#34: WG meeting time & Frequency](https://github.com/nodejs/i18n/issues/34)
* [#30: Translators for our WG meetings](https://github.com/nodejs/i18n/issues/20)
* Regular WG meeting issue owner (generate issues for our i18n WG meetings)?
* Regular WG Meeting minutes taker?

### Reports
* [#36: Initial i18n team meeting with Crowdin](https://github.com/nodejs/i18n/issues/36)
* ECMA-402 & Unicode Standards update (ICU, CLDR) - @srl295
* Anything else?

### Current Initiatives
* [#19: Audit of our l10n groups](https://github.com/nodejs/i18n/issues/19)
* Getting our i18n module going & what we can build right now (Outcome: generate an issue)
* Crowdin Integration (Outcome: generate an issue)

## Joining the meeting
* Video call link will be provided for participants a few minutes before the meeting starts
* A live YouTube stream link will be shared a few minutes before the meeting also.

### PR DESCRIPTION
**Pull request by RichardLitt**
This seems like the best place to put them.

This should close #37.

### CODE REVIEW

File: minutes/2018-03-06.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+WG Meeting minutes taker
+
+### Reports
+[#36: Initial i18n team meeting with Crowdin](https://github.com/nodejs/i18n/issues/36)
+
+ECMA-402 and Unicode Standards update (ICU, CLDR) - Steven Loomis
+ECMA 402 has had more frequent meetings such as https://github.com/tc39/ecma402/blob/master/meetings/agenda-2018-02-16.md  - many new features, lots of cooperation among implementers (including v8 stakeholders). 
+ICU 61 / CLDR 33 are finishing up shortly.  Expect an alpha PR 
+I found a couple of issues with running Node on older ICUs (including ICUs built into older systems  - https://github.com/nodejs/node/pull/19150 and there are others like it.
+identify which versionf of ICU works on node.  ( v59+ ) but should be 58 or even 57.
```
Comment (TiagoDanin) [inline minutes/2018-03-06.md]: Modify here: identify which **version** of ICU works on node.

File: minutes/2018-03-06.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+- @volem (Volkan Nazmi Metin)
+- @zeke (Zeke Sikelianos)
+
+## Present Members
+- @obensource (Ben Michel)
+- @bnb (Tierney Cyren)
+- @maddhruv (Dhruv Jain)
+- Adam Miller (Adam Miller)
+- @zeke  (Zeke Sikelianos)
+- Jacob Groundwater
```
Comment (zeke) [inline minutes/2018-03-06.md]: Jacob is @groundwater

File: minutes/2018-03-06.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+- @RichardLitt (Richard Littauer)
+- Franz
```
Comment (zeke) [inline minutes/2018-03-06.md]: Franz is actually "Juande". Handle is @FranzDeCopenhague

**Comment by RichardLitt** [comment]
Feel free to change things in the future in the doc. I just wanted a record so we don't rely on lost Google Docs.

**Pull request merged**